subsystem/src/main/java/org/wildfly/extension/elytron/tls/subsystem/SecurityPropertiesWriteHandler.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2017, Red Hat, Inc., and individual contributors as indicated
+ * by the @authors tag.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.extension.elytron.tls.subsystem;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jboss.as.controller.AbstractWriteAttributeHandler;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.controller.PropertiesAttributeDefinition;
+import org.jboss.dmr.ModelNode;
+import org.jboss.msc.service.ServiceController;
+import org.jboss.msc.service.ServiceRegistry;
+
+/**
+ * @author Tomaz Cerar (c) 2017 Red Hat Inc.
+ */
+class SecurityPropertiesWriteHandler extends AbstractWriteAttributeHandler {
```

Comment:
Just curious, since `security-properties` isn't part of the `tls` configuration in the original subsystem, just wondering about its inclusion here.

Reply:
Same as the previous comments on cred stores. should be able to remove once dependencies are updated and configured properly.